   ml_ops/sm-datazone_import/README.md

```diff
@@ -31,11 +32,43 @@ Run the script and follow the instructions.
 ```bash
 python import-sagemaker-domain.py \
     --region REGION \
-    --federation-role ARN_OF_FEDERATION_ROLE \
     --account-id ACCOUNTID
 ```
 
 ### Additional Configuration
 
 - SageMaker execution roles need DataZone API permissions in order for the Assets UI to function. See [DataZoneUserPolicy.json](./resources/DataZoneUserPolicy.json) for an example.
-- Ensure the DataZone Domain trusts SageMaker. In the AWS DataZone console navigate to Domain details and select the "Trusted services".
\ No newline at end of file
+- Ensure the DataZone Domain trusts SageMaker. In the AWS DataZone console navigate to Domain details and select the "Trusted services".
```

Comment:
note: need to double check that this "trusted services" tab still exists after NS launch